packages/components/src/index.native.js

```diff
@@ -46,6 +46,7 @@ export { default as withFocusOutside } from './higher-order/with-focus-outside';
 export { default as withFocusReturn } from './higher-order/with-focus-return';
 export { default as withNotices } from './higher-order/with-notices';
 export { default as withSpokenMessages } from './higher-order/with-spoken-messages';
+export * from './text';
```

Comment:
@ItsJonQ something to align with your other PR where you change the way it's exported for web.

Reply:
Ah yes! Thanks for the heads up.

I'm doing something similar (or related) in this PR:
https://github.com/WordPress/gutenberg/pull/21088